lib/npg_pipeline/pluggable.pm

```diff
@@ -701,32 +702,55 @@ sub _run_spider {
   return;
 }
 
-#####
-# Attempts to create in the analysis directory a hard link to
-# or, if unsuccessful, a copy of the log file of the pipeline.
-# All errors are captured. Best effort.
-#
-sub _create_log_link {
-  my $self = shift;
+# Attempts to link the pipeline log file to the analysis directory
+sub _copy_log_to_analysis_dir {
+  my ($self) = @_;
+  $self->_tolerant_persist_file_to_analysis_dir(
+    $self->log_file_path, $self->log_file_name
+  );
+  return;
+}
 
-  (-e $self->analysis_path) or return;
+sub _save_product_conf_to_analysis_dir {
+  my ($self) = @_;
+  # First check for unusual file names:
+  my ($filename, $dirs, $suffix) = fileparse(
+    $self->product_conf_file_path,
+    qr/[.][^.]*/xms
+  );
+  if ($filename ne 'product_release') {
```

Comment:
<($filename !~ /product_release/)> ?
Otherwise `product_release-mg.yml` is copied to `product_release_product_release-mg_20210318-171328-425726136.yml`